src/status_im/ui/components/bottom_sheet/view.cljs

```diff
@@ -90,75 +83,116 @@
      :onPanResponderRelease       (on-release opts)
      :onPanResponderTerminate     (on-release opts)})))
 
-(defn pan-handlers [pan-responder]
+(defn- pan-handlers [pan-responder]
   (js->clj (.-panHandlers pan-responder)))
 
-(defn- bottom-sheet-view
-  [{:keys [opacity-value bottom-value]}]
-  (reagent.core/create-class
-   {:component-did-mount
-    (fn []
-      (react/dismiss-keyboard!)
-      (animate-panel-open opacity-value bottom-value))
-    :reagent-render
-    (fn [{:keys [opacity-value bottom-value height
-                 content on-cancel disable-drag?]
-          :or   {on-cancel #(re-frame/dispatch [:bottom-sheet/hide])}
-          :as   opts}]
-      [react/keyboard-avoiding-view {:style styles/container}
-       [react/touchable-highlight
-        {:on-press #(cancel opts on-cancel)
-         :style    styles/container}
+(defn- on-open [{:keys [bottom-value internal opacity-value]}]
```

Comment:
probably its better to use asterisks for atoms , like `*internal*` ?

Reply:
not only here

Reply:
Aren't' muffles traditionally used only for dynamic vars and not atoms?

Reply:
@yenda yeah probably, do you have any better idea? because it would be better if we could recognize atoms in code

Reply:
Maybe an "-atom" suffixe, most of the time this doesn't happen as the atoms aren't passed by reference but by value instead so that could be a solution